app/Models/Matter.php

```diff
@@ -164,6 +190,15 @@ public function annuityAgents()
             ->wherePivot('role', 'ANN');
     }
 
+    public function sharedAnnuityAgents()
+    {
+        return $this->belongsToMany(Actor::class, 'matter_actor_lnk', 'matter_id', 'actor_id', 'container_id')
+            ->using(ActorPivot::class)
+            ->withPivot('role', 'display_order', 'shared', 'actor_ref')
+            ->wherePivot('role', 'ANN')
+            ->wherePivot('shared', 1);
+    }
+
     public function responsibles(): \Illuminate\Database\Eloquent\Relations\HasMany
```

Comment:
There is only one responsible in a matter, specifically identified in the matter's `responsible` field.